scripts/contractInteraction/mainnet_contracts.json

```diff
@@ -30,7 +30,14 @@
     "RIF": "0x2acc95758f8b5f583470ba265eb685a8f45fc9d5",
     "MYNT": "0x2e6B1d146064613E8f521Eb3c6e65070af964EbB",
     "swapNetwork": "0x98aCE08D2b759a265ae326F010496bcD63C15afc",
+    "ammSwapSettings": "0x3aB00BEFDd7Bfc0667DE6483D2D3b2F9A74AF2da",
+    "ammContractRegistry": "0x46EBC03EF2277308BdB106a73d11c65109C4B89B",
+    "ammOracleWhitelist": "0x8e75774Ef928cE730255AB594dD1b9F0a725a56b",
     "ConverterRegistry": "0x31A0F8400c75d52FdB413372233F28E3bdFB1c06",
+    "ConverterFactory": "0xcF46f24423B8da97E2c06B41df28163D55e80935",
+    "ConversionPathFinder": "0xcF46f24423B8da97E2c06B41df28163D55e80935",
```

Comment:
wrong address - the same as ConverterFactory

Reply:
fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/450/commits/0e0566114d689e3c539e0d9ee073d202c8366999)